### PR TITLE
OH GOD I HAVE NO IDEA WHAT I'M DOING

### PR DESCRIPTION
**Pull request by Cuboos**
THIS WAS GOOF'S IDEA, I SWEAR, …BLAME HIM. SPARE ME. BE GENTILE SENPAI.

### CODE REVIEW

File: html/browser/chroma_green.css [hunk 1]
```diff
@@ -0,0 +1,306 @@
+body 
+{ 
+	padding: 0;
```
Comment (Iamgoofball) [inline html/browser/chroma_green.css]: font should be courier new

**Comment by Jordie0608** [comment]
I can pretty much tell from the files what this does but you should really add a description =P

File: html/browser/chroma_green.css (hunk 1 above)
Comment (Cuboos) [inline html/browser/chroma_green.css]: GOD DAMN IT GOOF BALL SHUT UP ABOUT THE FONT.

**Comment by Cuboos** [comment]
Can someone add a Do Not Merge tag? it's not finished, goof suggested i upload it to github in order to get help.

**Comment by Cuboos** [comment]
Thanks

**Comment by Aranclanos** [comment]
explain what this is

**Comment by TrustyGun** [comment]
it looks like he is trying to add images to The Orion Trail's game window for each event

**Comment by xxalpha** [comment]
I thought the arcade machines had sounds already.

**Comment by Xhuis** [comment]
@xxalpha Only the "fight" arcade machines have sounds.

**Comment by Bawhoppen** [comment]
Orion Trail has some sounds

**Comment by Cuboos** [comment]
They're all reused sounds though, they don't sound like game sounds.

File: html/typing2.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+inserChar = '|'
```
Comment (Iamgoofball) [inline html/typing2.js]: can any JAVASCRIPT GURUS show up and help us get this memery working

we want the console text effect on all text in the arcade game

**Comment by Cheridan** [comment]
next gen :open_mouth:

**Comment by astralenigma** [comment]
Hey cuboos despite the game screaming about the JavaScript errors everytime it updates content what's wrong?

**Comment by Cuboos** [comment]
Just the JavaScript, everything else runs fine. Also the project isn't complete, i have more graphics to be made. 

Honestly i can live without the typing, it's just @Iamgoofball that's really adamant about it.

File: code/game/machinery/computer/arcade.dm [hunk 3]
```diff
@@ -427,14 +429,17 @@
 			gameStatus = ORION_STATUS_START
 			name = "The Orion Trail"
 			desc = "Learn how our ancestors got to Orion, and have fun in the process!"
+		playsound(src.loc, 'sound/arcade/orion/Ori_fail.ogg', 50, 0, extrarange = -3, falloff = 10)
```
Comment (duncathan) [inline code/game/machinery/computer/arcade.dm]: orion trail? more like orion **fail** ohhhhhh

Comment (Cuboos) [inline code/game/machinery/computer/arcade.dm]: Ban @duncathan

**Comment by astralenigma** [comment]
What exactly do you need the JavaScript to be able to do with the typing?

**Comment by Cuboos** [comment]
@Iamgoofball wanted the flavor text to be typed out. I could recreate the effect with a .gif, which may take up more space, but would be less glitchy and a lot less difficult to implement.

File: code/game/machinery/computer/arcade.dm [hunk 4]
```diff
@@ -445,13 +450,23 @@
...
+		//dat = "<center><h2>The Orion Trail</h2></center>" Commenting out for a test right now
+		dat = "<br><center><img src=shipimg></center>" //LIFE LIKE TEXTURES
 		dat += "<br><center><h3>Experience the journey of your ancestors!</h3></center><br><br>"
 		dat += "<center><b><a href='byond://?src=\ref[src];newgame=1'>New Game</a></b></center>"
 		dat += "<P ALIGN=Right><a href='byond://?src=\ref[src];close=1'>Close</a></P>"
 	var/datum/browser/popup = new(user, "arcade", "The Orion Trail",400,700)
 	popup.set_content(dat)
+	popup.add_stylesheet("arcade", 'html/browser/chroma_green.css')
+	popup.add_script("arcade","libraries.min.js")
+	popup.add_script("arcade",'typing2.js')
```
Comment (astralenigma) [inline code/game/machinery/computer/arcade.dm]: Do you have these Scripts or do you need them made?

Comment (Cuboos) [inline code/game/machinery/computer/arcade.dm]: Yeah... they're in the PR.

Comment (astralenigma) [inline code/game/machinery/computer/arcade.dm]: Only found the Typing2.js

**Comment by astralenigma** [comment]
@Iamgoofball What exactly do you mean by flavour text or better what isn't flavour text?

File: code/game/machinery/computer/arcade.dm (hunk 4 above)
Comment (Cuboos) [inline code/game/machinery/computer/arcade.dm]: The libraries.min.js is already in the files, the crew monitor uses JQurry.

**Comment by Iamgoofball** [comment]
The text describing the current scenario.

**Comment by Cuboos** [comment]
I still have no idea what i'm doing...

File: html/typing.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
+var message;
+pos=0;
+var i = Math.floor(Math.random()*16);
```
Comment (astralenigma) [inline html/typing.js]: Where do you use this var?

Comment (RemieRichards) [inline html/typing.js]: @astralenigma ```maxlength = message.length+1;``` in ```writemsg(msg)```

**Comment by Razharas** [comment]
Ok i pronounce this dead

**Pull request closed without merging**